src/component/parser/json/converter/AbstractConverter.ts

```diff
@@ -37,17 +34,7 @@ export function ensureIsArray<T>(elements: (T | string)[] | T | string, acceptEm
   return returnedArray.filter(value => value);
 }
 
-@JsonConverter
-export abstract class AbstractConverter<T> implements JsonCustomConvert<T> {
-  // TODO find a way to inject JsonConvert, see #110
-  protected readonly jsonConvert = JsonConvertConfig.jsonConvert();
-
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars,@typescript-eslint/no-explicit-any
-  serialize(data: T): any {
-    // TODO throw exception
-    console.error('Not implemented !!');
-  }
-
+export abstract class AbstractConverter<T> {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  abstract deserialize(data: any): T;
+  public abstract deserialize(data: any): T;
```

Comment:
:question: are you sure we need to add this? we use implicit public in the rest of the code

Reply:
Indeed